specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/examples/SuccessfulAnalyzeTextJobSubmitRequest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2021-11-01-preview",
+    "Endpoint": "{Endpoint}",
+    "jobId": "{Job ID}",
+    "input": {
+      "displayName": "Extracting Location & US Region",
```

Comment:
I believe the original example included Custom Entities task which is why it has US regions mentioned. 
Can we reinstate a custom entities task here?

Reply:
Ok, I have added a custom entities task to the input and response examples.